build/linux/installer/scripts/tomlparser-agent-config.rb

```diff
@@ -55,6 +55,12 @@
 @podsEmitStreamBatchSizeMin = 50
 @nodesEmitStreamBatchSizeMin = 50
 
+# configmap settings related fbit config
+@fbitFlushIntervalSecsSecs = 0
```

Comment:
Should be @fbitFlushIntervalSecs I believe

Reply:
> Should be @fbitFlushIntervalSecs I believe

> yep, your right. :) fixed. thanks for catching.